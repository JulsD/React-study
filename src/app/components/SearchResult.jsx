import React from 'react';
import MoviesList from './MoviesList';

class SearchResult extends React.Component {

  constructor(props) {
    super(props);

  }

  render() {
    const movies = this.props.movies;

    let searchResultBody = null;
    if (movies) {
      searchResultBody = <MoviesList movies={movies}/>;
    } else {
      searchResultBody = 'No films found';
    }
    return (
      <div className="search-resuls">
        {searchResultBody}
      </div>
    );
  }

}

export default SearchResult;
