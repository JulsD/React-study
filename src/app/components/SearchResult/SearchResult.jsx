import React from 'react';

import Wrapper from './../Wrapper';

class SearchResult extends React.Component {

  constructor(props) {
    super(props);

  }

  render() {
    return (
      <div>
        <Wrapper>
          {this.props.children}
        </Wrapper>
      </div>
    );
  }

}

export default SearchResult;
