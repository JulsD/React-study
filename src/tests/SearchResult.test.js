import React from 'react';
import SearchResult from '../app/components/SearchResult';

test('SearchResult should render a <nav>', () => {
  const wrapper = shallow(
    <SearchResult />
  );
  expect(wrapper).toMatchSnapshot();
});
