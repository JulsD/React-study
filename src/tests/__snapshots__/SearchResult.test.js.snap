// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchResult should render a <nav> 1`] = `
<div>
  <Wrapper />
</div>
`;
